sspl_test/run_qa_test.sh

```diff
@@ -58,7 +68,10 @@ pre_requisites()
         # Backing up original persistence data
         $sudo rm -rf /var/$PRODUCT_FAMILY/sspl/orig-data
         $sudo mkdir -p /var/$PRODUCT_FAMILY/sspl/orig-data
-        $sudo find /var/$PRODUCT_FAMILY/sspl -maxdepth 2 -type d -path "/var/$PRODUCT_FAMILY/sspl/data/*" -not -name 'iem'  -exec bash -c 'mv -f ${0} ${0/data/orig-data}/' {} \;
+        $sudo find "/var/$PRODUCT_FAMILY/sspl" -maxdepth 2 \
+            -type d -path "/var/$PRODUCT_FAMILY/sspl/data/*" \
+            -not \( -name 'iem' -o  -name 'coverage' \)  \
+            -exec bash -c 'mv -f ${0} ${0/data/orig-data}/' {} \;
```

Comment:
single-quote is needed, I am not getting expected results with double-quotes.

Reply:
Yes, single quotes is required to avoid expansion of echo expression.

Reply:
ack